pkg/api/v1/types.go

```diff
@@ -2251,6 +2259,9 @@ type Event struct {
 
 	// The number of times this event has occurred.
 	Count int `json:"count,omitempty"`
+
+	// Type of this event (Normal, Warning), new types could be added in the future
+	Type string `json:"type,omitempty"`
```

Comment:
I guess this name was @bgrant0607's idea-- I still think "type" is an ambiguous name.